doc/radosgw/multisite.rst

```diff
@@ -1170,26 +1170,26 @@ a minimum, specifying the required settings:
    object data. Set to ``default-placement`` by default. It is  also possible
    to set a per-user default placement in the user info for each user.
 
-To set a zonegroup, create a JSON object that contains the required fields,
-save the object to a file (e.g., ``zonegroup.json``), and run the following
-command:
+Setting a Zonegroup - Procedure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. prompt:: bash #
+#. To set a zonegroup, create a JSON object that contains the required fields,
+   save the object to a file (for example, ``zonegroup.json``), and run the
+   following command:
+
+   .. prompt:: bash #
    
-   radosgw-admin zonegroup set --infile zonegroup.json
+      radosgw-admin zonegroup set --infile zonegroup.json
 
-Where ``zonegroup.json`` is the JSON file you created.
+   Where ``zonegroup.json`` is the JSON file you created.
 
-.. important:: The ``default`` zonegroup ``is_master`` setting is ``true`` by
-   default. If you create a new zonegroup and want to make it the master
-   zonegroup, you must either set the ``default`` zonegroup ``is_master``
-   setting to ``false``, or delete the ``default`` zonegroup.
+   .. important:: The ``default`` zonegroup ``is_master`` setting is ``true`` by default. If you create a new zonegroup and want to make it the master zonegroup, you must either set the ``default`` zonegroup ``is_master`` setting to ``false``, or delete the ``default`` zonegroup.
```

Comment:
s/new/additional

As written this is confusing if there are no existing instances.

Reply:
You're not wrong. I'll fix it in this PR.